### DISCUSSION TITLE
Suggesting Vocab for higher accuracy

### DISCUSSION THREAD

**Discussion by batmurph** [openai/whisper#328] [Q&A] [answered]
Is it possible to suggest lines of text to "trend" it toward certain vocabulary for a transcription. For example, a page of a book sampled before running so that it might adopt similar vocab when in instances where there is words that may share a sound but not meaning(or kanji for jp)
Similar to the "punctuation mode" discussed in #204

**Comment by jianfch** [comment, marked as answer]
As answered in #204, you can use `--initial_prompt` for cli or `prompt` for lower-level python. How `prompt` works is explained in the first half of this [answer](https://github.com/openai/whisper/discussions/117#discussioncomment-3727051).
So you can try to use the text of that page for `prompt`.

Note: `prompt` is use in the lower-level access methods such as `DecodingOptions()`, but `initial_prompt` is still used for `transcribe()`

**Reply by batmurph** [reply]
Great! I was unaware that prompt could also work in ways other than punctuation. Thanks for the reply!

**Reply by Abdurrafey-Siddiqui** [reply]
Hey can you let me know how you did it? Cause when i tried it would probably work for maybe the first 30 seconds then not the rest of the audio. Also what if you want to do it for multiple words and can the vocabs that i add manually be used globally (ie; every time I transcribe an audio?) or must I include this command all the time?
